dependencies/url/validator.go

```diff
@@ -55,6 +55,7 @@ var privateIPBlocks []*net.IPNet
 
 func init() {
 	for _, cidr := range []string{
+		"0.0.0.0/16",
```

Comment:
Why a subnet of `/16` should it be `/32`? From my quick testing only the address `0.0.0.0` not `0.0.0.1` or any other combination with zeros is an alias for localhost. Also according to https://en.wikipedia.org/wiki/0.0.0.0 it seems that 0.0.0.0 itself is the special address not a range of addresses.

Reply:
Ah yes! I suspect you're right and this should be a single address.